doc/user/content/free-trial-faqs.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Free trial FAQs"
+description: "Answers to frequency asked questions about Materialize free trials"
+menu:
+  main:
+    parent: "quickstarts"
+    weight: 15
+---
+
+When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product without worrying about initial costs. This page answers some frequently asked questions about free trials.
```

Comment:
```suggestion
When you [sign up for Materialize](https://materialize.com/register/), you get a free trial account so you can explore the product and start building! This page answers some frequently asked questions about free trials.
```

Reply:
> so you can explore the product without worrying about initial costs.

This makes me iffy. It should be focused on the value, rather than the cost.

Reply:
Changed.